src/lib/loaders/loaders_with_authentication/impulse.ts

```diff
@@ -50,5 +50,10 @@ export default (accessToken, userID, opts) => {
       },
       { method: "POST" }
     ),
+    conversationCreateConversationOrderLoader: impulseLoader(
```

Comment:
i feel like `createConversationOrderLoader` makes more sense, but I see all these impulse ones have the prefix `conversation-`.

Reply:
happy to break this convention if you say it's ok!